tests/unit/utils/google-maps-test.js

```diff
@@ -0,0 +1,10 @@
+import googleMaps from 'super-rentals/utils/google-maps';
```

Comment:
Not sure I want to unit test this because of the google network requests . I'll do some research.